docker/ci/application.yml

```diff
@@ -0,0 +1,95 @@
+# NOTE: when changing this file you must rerun:
+# - docker compose --file docker-compose-local.yml build
+
+armadillo:
+  # DO NOT CHANGE
+  docker-management-enabled: false
+  # DO NOT CHANGE
+  docker-run-in-container: true
+
+  # when running the R containers from a docker-compose.yml they get prefixes based on the directory name of the
+  # docker-compose.yml file ie armadillo-dev-" + profileName + "-1". Same goes for Armadillo ie armadillo-dev-armadillo-1"
+  container-prefix: 'dev-'
```

Comment:
This is weird as we should have 'ci-' as the containers are named after `cwd` when `docker compose up`